priv/gettext/es/LC_MESSAGES/endgame.po

```diff
@@ -79,248 +79,248 @@ msgstr ""
 #: lib/listudy_web/controllers/endgame_controller.ex:355 lib/listudy_web/controllers/endgame_controller.ex:380
 #: lib/listudy_web/controllers/endgame_controller.ex:408 lib/listudy_web/controllers/endgame_controller.ex:462
 msgid "From the book \"Chess Studies, Or, Endings of Games\"."
-msgstr ""
+msgstr "Del libro \"Estudios, o, finales\"."
 
 #, elixir-format
 #: lib/listudy_web/controllers/endgame_controller.ex:127
 msgid "King and Pawn Endgames"
-msgstr ""
+msgstr "Finales con Rey y Peón"
 
 #, elixir-format
 #: lib/listudy_web/controllers/endgame_controller.ex:421
 msgid "King and Queen against King and Rook"
-msgstr ""
+msgstr "Rey y Dama contra Rey y Torre"
 
 #, elixir-format
 #: lib/listudy_web/controllers/endgame_controller.ex:230
 msgid "Knights, Bishops and Pawns"
-msgstr ""
+msgstr "Caballos, Alfiles y Peones"
 
 #, elixir-format
 #: lib/listudy_web/controllers/endgame_controller.ex:61
 msgid "Learn the basic checkmating techniques."
-msgstr ""
+msgstr "Aprende las técnicas básicas para dar jaque mate."
 
 #, elixir-format
 #: lib/listudy_web/controllers/endgame_controller.ex:327
 msgid "Minor pieces against Rook endgames."
-msgstr ""
+msgstr "Finales de piezas menores contra Torre"
 
 #, elixir-format
 #: lib/listudy_web/controllers/endgame_controller.ex:326
-msgid "Minor pirces against Rook"
-msgstr ""
+msgid "Minor pieces against Rook"
```

Comment:
msgid's should not get edited for typos. Please undo this change and I will fix the typo in the file.